pkg/operator/ceph/cluster/cleanup.go

```diff
@@ -198,11 +199,13 @@ func (c *ClusterController) waitForCephDaemonCleanUp(stopCleanupCh chan struct{}
 
 			logger.Debugf("waiting for ceph daemons in cluster %q to be cleaned up. Retrying in %q",
 				cluster.Namespace, retryInterval.String())
-			break
+			break out
```

Comment:
It looks like the intention is to wait for all ceph daemons to be cleaned up and return on line 197. If we change this break statement, we will no longer be waiting for all the daemons to be cleaned up, right?

Reply:
Ok but what is the intent for this `break` then? Should we just remove it?

Reply:
It will break out of the `select` statement, then the `for` loop will cause it to wait again in the next instance of the `select` statement for the retryInterval, right?

Reply:
Hum ok but why do we need to break the `select`? Can't we just leave it empty then? Isn't that the same thing?
If the goal is not to break out of the `for` then I don't understand why we need it.

Reply:
Oh right, with golang the `break` is implied if it is not specified, so the behavior would be the same with or without it. But were you trying to fix some issue?